packages/concerto-core/test/introspect/introspector.js

```diff
@@ -80,6 +81,32 @@ describe('Introspector', () => {
             const introspector = new Introspector(modelManager);
             introspector.getClassDeclaration('org.acme.base.Person').should.not.be.null;
         });
+
+        it('should be able to handle the aliased imported types', () => {
+            // create and populate the ModelManager with a model file
+            const modelManager = new ModelManager();
+            Util.addComposerModel(modelManager);
+            modelManager.enableAliasedType = true;
+            modelManager.should.not.be.null;
+
+            const model1 = `
+            namespace org.saluja.ext
```

Comment:
`saluja.org` seems to be a real domain. Can you please change to `com.example` or `org.acme` ?

Reply:
Sure will fix this